src/index.css

```diff
@@ -0,0 +1,5 @@
+body {
```

Comment:
### Nitpick

This is a duplicate of what's contained in the previous stylesheet. One of them should be deleted.